MAUI/NumericEntry/UpDown-Button.md

```diff
@@ -69,6 +69,143 @@ sfNumericEntry.UpDownPlacementMode = NumericEntryUpDownPlacementMode.Inline;
 
 ![UpDown Placement in .NET MAUI NumericEntry](UpDownButton_images/UpDownButtonPlacement.gif)
 
+# UpDown button customization
+
+## UpDown button color
+
+You can customize the `NumericEntry` control button color by using the `UpDownButtonColor` property.
```

Comment:
Customize the `NumericEntry` control button color by using the `UpDownButtonColor` property.